src/asqi/validation.py

```diff
@@ -817,3 +824,60 @@ def validate_workflow_configurations(
         errors.extend(validate_manifests_against_tests(suite, systems, manifests))
 
     return errors
+
+
+def validate_score_cards_reports(
+    suite: SuiteConfig, manifests: Dict[str, Manifest], score_cards: List[ScoreCard]
+) -> List[str]:
+    """
+    Validate score cards configurations against test suite and manifests.
+
+    Notes:
+        - No duplicate report names within each indicator's `display_reports`
+        - All requested `display_reports` exist in the corresponding test's manifest
+
+    Args:
+        suite: Test suite configuration
+        manifests: Dictionary mapping image names to their manifests
+        score_cards: List of score card configurations
+
+    Returns:
+        List of validation error messages
+    """
+    errors = []
+    test_id_to_image = {test.id: test.image for test in suite.test_suite}
+
+    for score_card in score_cards:
+        for indicator in score_card.indicators:
+            if isinstance(indicator, AuditScoreCardIndicator):
+                continue
+            test_id = indicator.apply_to.test_id
+            image = test_id_to_image[test_id]
```

Comment:
If a score card references a test_id that doesn't exist in the suite, it will crash the application - let's throw an error instead